src/components/cards/CardAvatar.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const CardAvatar = ({ ownerProfileUrl, avatarUrl, className }) => {
+  let cardAvatarClassName = 'card-avatar ' + className;
+
+  return (
+    <div className={cardAvatarClassName}>
+      <a href={ownerProfileUrl}>
```

Comment:
You'll have to update this to 
```suggestion
      <Link to={`/members/${memberId}`}>
```